dali-mqtt-daemon.py

```diff
@@ -289,6 +290,20 @@ def create_mqtt_client(
     }
 
     exception_raised = False
+    all_supported_log_levels = {
+        "critical": logging.CRITICAL,
+        "error": logging.ERROR,
+        "warning": logging.WARNING,
+        "info": logging.INFO,
+        "debug": logging.DEBUG,
+    }
```

Comment:
you can use this list in the add_argument() method to validate the input, check the way I do it with the dali drivers

Reply:
I moved this dictionary to consts.py, and used `choices` in argparse